CHANGELOG.md

```diff
@@ -2,6 +2,73 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - 2019-07-??
+ - New Features
+   - `pysatData` directory created in user's home directory if no directory specified
+   - Added preliminary support for `xarray` to the `instrument` object
+   - Support for `today`, `tomorrow`, and `yesterday` as datetime objects
+   - Added `model_utils`, featuring preliminary support for data-model comparison
+   - Added support for 1d median in seasonal averages
+   - Added routine to convert from kp to Ap
+   - Added `pyglow` integration support for python 3.x
+   - Improves input handling for datetime parameters that are more precise than just year, month, and day, where appropriate
+   - Added merging routines to allow combination of measured and forecasted Kp and F10.7 indexes into a single instrument object
+ - Code restructure
+   - Moved instrument templates and methods to subdirectories
+   - Moved utils into multiple subdirectories to aid with organization
+ - Instrument Updates
+   - NASA CDAWeb download now uses https protocol rather than FTP
+   - `_instrument.py` supports xarray
+   - Support for listing files from remote server
+   - COSMIC RO data unified into single instrument object
+   - Added support for DEMETER IAP
+   - Added support for DMSP IVM Level 1 data
```

Comment:
Should be level-2. It is geophysical data.

Reply:
May also be worth noting it uses madrigal methods pysat support along with openMadrigal.

Reply:
files changes show Level 1 is new data.  May be a typo.  Added comment in #247.

Reply:
My guess is a typo, I've always encountered geophysical data at level-2.

I checked with Madrigal. The writeup by Marc lists level-2 as geophysical and essentially perfect. It calls the data level-1 but only because the standard is set very high. I recommend switching it to level-2. By default we apply cleaning flags thus it is as good as it gets when it comes to DMSP ion drifts.

Reply:
fixed.